src/SDKs/SqlManagement/Sql.Tests/SyncMemberScenarioTests.cs

```diff
@@ -0,0 +1,125 @@
+                SyncGroup createSyncGroup = sqlClient.SyncGroups.CreateOrUpdate(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName, new SyncGroup
+                {
+                    Interval = -1, // Manual
+                    ConflictResolutionPolicy = SyncConflictResolutionPolicy.MemberWin,
+                    SyncDatabaseId = syncDatabase.Id,
+                    HubDatabaseUserName = SqlManagementTestUtilities.DefaultLogin,
+                    HubDatabasePassword = SqlManagementTestUtilities.DefaultPassword
+                });
+                Assert.NotNull(createSyncGroup);
+
+                #region Azure SQL database member
+
+                // Create an Azure SQL database member
+                string syncMemberName = SqlManagementTestUtilities.GenerateName(testPrefix + "azsyncmember");
+                string syncMemberDirection = SyncDirection.OneWayMemberToHub;
+                string syncMemberDatabaseType = SyncMemberDbType.AzureSqlDatabase;
+                SyncMember createSyncMember = sqlClient.SyncMembers.CreateOrUpdate(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName, syncMemberName, new SyncMember
+                {
+                    SyncDirection = syncMemberDirection,
+                    DatabaseType = syncMemberDatabaseType,
+                    DatabaseName = memberDatabaseName,
+                    ServerName = server.Name,
+                    UserName = SqlManagementTestUtilities.DefaultLogin,
+                    Password = SqlManagementTestUtilities.DefaultPassword,
+                });
+                Assert.NotNull(createSyncMember);
+                Assert.Equal(syncMemberDirection, createSyncMember.SyncDirection);
+                Assert.Equal(syncMemberDatabaseType, createSyncMember.DatabaseType);
+                Assert.Equal(memberDatabaseName, createSyncMember.DatabaseName);
+                Assert.Equal(server.Name, createSyncMember.ServerName);
+
+                // Get Azure SQL database member
+                SyncMember getSyncMember = sqlClient.SyncMembers.Get(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName, syncMemberName);
+                Assert.NotNull(getSyncMember);
+                Assert.Equal(syncMemberDirection, getSyncMember.SyncDirection);
+                Assert.Equal(syncMemberDatabaseType, getSyncMember.DatabaseType);
+                Assert.Equal(memberDatabaseName, getSyncMember.DatabaseName);
+                Assert.Equal(server.Name, getSyncMember.ServerName);
+
+                // List sync members
+                IPage<SyncMember> listSyncMembers = sqlClient.SyncMembers.ListBySyncGroup(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName);
+                Assert.NotNull(listSyncMembers);
+                Assert.Equal(1, listSyncMembers.Count());
+                Assert.Equal(syncMemberName, listSyncMembers.FirstOrDefault()?.Name);
```

Comment:
nit: already checked that there is one, so this can be `listSyncMembers.Single().Name`